cmd/mapper.go

```diff
@@ -23,10 +23,19 @@ func main() {
 
 	token := ""
 	cli.StringFlag("secure-token", "API token for Sysdig Secure (env: SECURE_API_TOKEN)", &token)
+
+	// No Cli flag for the token, assign from the env var
 	if len(token) == 0 {
 		token = os.Getenv("SECURE_API_TOKEN")
 	}
 
+	// Missing token cli flag or environment variable
+	_, ok := os.LookupEnv("SECURE_API_TOKEN")
+	if !ok {
```

Comment:
I think you need to move this under the `if len(token) == 0 {}` block above. Right now this will fail any time the ENV var is missing, even if the token is provided via flag.

Something like this perhaps?
```go
	// No Cli flag for the token, assign from the env var
	if len(token) == 0 {
		// Missing token cli flag or environment variable
		var ok bool
		token, ok = os.LookupEnv("SECURE_API_TOKEN")
		if !ok {
			fmt.Println("TOKEN NOT FOUND - Please ensure either the \"SECURE_API_TOKEN\" environment variable or the \"secure-token\" cli flag is set.")
			os.Exit(1)
		}
	}
```

Otherwise, this is a great improvement from the existing behavior of blindly passing the 401 error from the API. 😄